federation/plugin/pkg/admission/schedulingpolicy/admission.go

```diff
@@ -37,21 +39,34 @@ const (
 	pluginName               = "SchedulingPolicy"
 	configKey                = "schedulingPolicy"
 	policyConfigMapNamespace = "kube-federation-scheduling-policy"
-	minRetryBackoff          = time.Millisecond
-	maxRetryBackoff          = time.Second * 60
-	defaultRetryBackoff      = time.Millisecond * 100
+
+	// Lower and upper bounds on policy engine query retry delays. The actual
+	// backoff implementation is handled by k8s.io/apiserver/pkg/util/webhook.
+	// The defaultRetryBackoff is used if the admission controller
+	// configuration does not specify one.
+	minRetryBackoff     = time.Millisecond
+	maxRetryBackoff     = time.Second * 60
+	defaultRetryBackoff = time.Millisecond * 100
 )
 
 type admissionConfig struct {
-	Kubeconfig   string        `json:"kubeconfig"`
-	RetryBackoff time.Duration `json:"retryBackoff"`
+	Kubeconfig                   string        `json:"kubeconfig"`
+	RetryBackoff                 time.Duration `json:"retryBackoff"`
+	FailOpenIfPolicyDoesNotExist bool          `json:"failOpenIfPolicyDoesNotExist"`
 }
 
 type admissionController struct {
 	*admission.Handler
 	policyEngineClient       *rest.RESTClient            // client to communicate with policy engine
 	policyEngineRetryBackoff time.Duration               // backoff for policy engine queries
-	client                   internalclientset.Interface // client to communicate with federation-apiserve
+	client                   internalclientset.Interface // client to communicate with federation-apiserver
+
+	// controls whether policy engine is queried when policy has not been
```

Comment:
I dont think we need this flag.
Even for testing, admin should create the policies in configmap.

If there is no configmap (i.e policies not defined), then admission controller can just return,
If there is a configmap, then admission controller should call the policy engine and fail in case of an error.